### PR TITLE
Jvthunder vgg code

### PR DESCRIPTION
**Pull request by JvThunder**
This is my initial pytorch code for vgg19.

### CODE REVIEW

File: course/2 Neural Network Architectures/2.1 Convolutional Neural Networks/2.1.2 Common Pre-Trained Models/vgg19.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            nn.ReLU(),
+            nn.Dropout(0.5),
+            nn.Linear(4096, num_classes),  # Output layer with 'num_classes' output units
+        )
+
+    def forward(self, x):
+        x = self.feature_extractor(x)  # Pass input through the feature extractor layers
+        x = x.view(x.size(0), -1)  # Flatten the output for the fully connected layers
+        x = self.classifier(x)  # Pass flattened output through the classifier layers
+        return x
```
Comment (sezan92) [inline course/2 Neural Network Architectures/2.1 Convolutional Neural Networks/2.1.2 Common Pre-Trained Models/vgg19.py]:
```suggestion
        return x
 
```

Comment (sezan92) [reply course/2 Neural Network Architectures/2.1 Convolutional Neural Networks/2.1.2 Common Pre-Trained Models/vgg19.py]: newline

**Review by sezan92 — COMMENTED** [review]
minor addition. check it . @ShamieCC please review

**Comment by sezan92** [comment]
@JvThunder after approval. you merge this PR to the main branch.

**Comment by JvThunder** [comment]
Okay, I have committed the changes, I will wait for @ShamieCC to review and then I will merge to main.

**Review by ShamieCC — APPROVED** [review]
You can merge to the main branch, am okay with what has been done already.

**Review by sezan92 — APPROVED** [review]

**Pull request merged**